Sources/SwiftTryCatch/SwiftTryCatch.m

```diff
@@ -0,0 +1,42 @@
+//
+//  SwiftTryCatch.h
+//
+//  Created by William Falcon on 10/10/14.
```

Comment:
This code was adapted from This code was adapted from https://github.com/williamFalcon/SwiftTryCatch/